podpac/core/coordinates/stacked_coordinates.py

```diff
@@ -429,7 +429,7 @@ def _transform(self, transformer):
             lat = coords[ilat]
             lon = coords[ilon]
             alt = coords[ialt]
-            tlat, tlon, talt = transformer.transform(lat.coordinates, lon.coordinates, alt.coordinates)
+            tlon, tlat, talt = transformer.transform(lon.coordinates, lat.coordinates, alt.coordinates)
```

Comment:
Should this also use `always_xy=True`?

Reply:
no, the Tranformer object is instantiated with `always_xy` so all child `transformer`s will have it.